gsmphys/radiation_clouds.F

```diff
@@ -2593,7 +2664,7 @@ subroutine progcld6                                               &
 #else
 #ifdef GFS_CLOUD_OVERLAP
       call cld_eff_rad (1, IX, 1, NLAY, slmsk, plyr*100,                &
-     &                  abs(plvl(:,1:NLAY)-plvl(:,2:NLAY+1))*100, tlyr, &
+     &                  abs(plvl(:,1:NLAY)-plvl(:,2:NLAY+1))*100, tylr, &
```

Comment:
I don't think you want this edit?

Reply:
Thanks -- indeed I didn't mean to change this.  I accidentally introduced this in porting these changes to this branch from my internal branch whose base code is slightly different.